jupyros/_version.py

```diff
@@ -1,43 +1,4 @@
-#############################################################################
-# Copyright (c) Wolf Vollprecht, QuantStack                                 #
-#                                                                           #
-# Distributed under the terms of the BSD 3-Clause License.                  #
-#                                                                           #
-# The full license is in the file LICENSE, distributed with this software.  #
-#############################################################################
-
-import json
-from pathlib import Path
-
-__all__ = ["__version__"]
-
-def _fetchVersion():
-    HERE = Path(__file__).parent.resolve()
-
-    for settings in HERE.rglob("package.json"):
-        try:
-            with settings.open() as f:
-                version = json.load(f)["version"]
-                return (
-                    version.replace("-alpha.", "a")
-                    .replace("-beta.", "b")
-                    .replace("-rc.", "rc")
-                )
-        except FileNotFoundError:
-            pass
-
-    raise FileNotFoundError(f"Could not find package.json under dir {HERE!s}")
-
-def _fetchJSVersion():
```

Comment:
Since this is removed, the import needs to change in *ros1/ros3d.py* to

```python
from ../_version import __version__

js_version = '^' + __version__
```